libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1716,14 +1716,15 @@ void GCS_MAVLINK::send_scaled_imu(uint8_t instance, void (*send_fn)(mavlink_chan
 // send data for barometer and airspeed sensors instances.  In the
 // case that we run out of instances of one before the other we send
 // the relevant fields as 0.
-void GCS_MAVLINK::send_scaled_pressure_instance(uint8_t instance, void (*send_fn)(mavlink_channel_t chan, uint32_t time_boot_ms, float press_abs, float press_diff, int16_t temperature))
+void GCS_MAVLINK::send_scaled_pressure_instance(uint8_t instance, void (*send_fn)(mavlink_channel_t chan, uint32_t time_boot_ms, float press_abs, float press_diff, int16_t temperature, int16_t temperature_press_diff))
 {
     const AP_Baro &barometer = AP::baro();
 
     bool have_data = false;
 
     float press_abs = 0.0f;
-    float temperature = 0.0f;
+    float temperature = 0.0f; // Absolute pressure temperature
+    float temperature_press_diff = 0.0f; // TODO: Differential pressure temperature
```

Comment:
This should be UINT16_MAX if we don't have it

Reply:
It's an extension though, so you can't tell if 0 is actually 0C or max... Usually with extensions you have to special case that 0 means unknown. (And if we don't have it a value of 0 is more amenable to packet trimming)

Reply:
Hum, your right, I was lead astray ;) , **https://github.com/mavlink/mavlink/pull/1429#issuecomment-663314776** Can change the comment to say 0 is the special case I guess.